README.md

```diff
@@ -34,8 +36,7 @@ Welcome to the `sauce-docs` repository. This website contains all technical docu
 ## Prerequisites
 
 -   [`git`](https://git-scm.com/downloads)
--   [Node.js](https://nodejs.org/en/download/) `version >= 14.15.0` or above (which can be checked by running node -v). You can use nvm for managing multiple Node versions on a single machine installed
--   [Yarn](https://yarnpkg.com/en/) `version >= 1.22.5` (which can be checked by running yarn version). Yarn is a performant package manager for JavaScript and replaces the npm client. It is not strictly necessary but highly encouraged.
```

Comment:
Note: Removed yarn because the examples and the pipelines actually use npm. Furthermore, npm is packaged with Node.js, so no separate installation required.